hangman.js

```diff
@@ -1 +1,28 @@
-console.log("welcome to hangman");
\ No newline at end of file
+var word = prompt("Choose your word").toLowerCase();
+var guesses = [];
+var incorrectGuessesLimit = 6;
+var letters = [];
+var countIncorrectGuesess = [];
+var incorrectGuesses = [];
+
+function convertWordToDashes(word){
+  var letters = word.split("");
+  var dashes = "";
+  for(var i = 0; i < letters.length; i++){
+    if(guesses.includes(letters[i])){
+      letters[i] = letters[i]
```

Comment:
Not sure I understand your intent on line 13, you are setting the variable equal to itself in this case